src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
+import StyledCrumb from "./crumb.style";
+
+export interface CrumbProps extends TagProps {
```

Comment:
You can just extend `LinkProps` here and `omit` the ones we don't want to expose